tests/kiwi_images_test/login_reboot.pm

```diff
@@ -0,0 +1,34 @@
+# SUSE's openQA tests
+#
+# Copyright © 2012-2019 SUSE LLC
```

Comment:
for new code isnt needed start in past :D

Reply:
You mean, I don't need the 2012 part?